projects/demo/src/app/problem/problem.component.ts

```diff
@@ -31,7 +31,7 @@ export class ProblemComponent {
         The age is required
       </div>
       <div *ngIf="form.get('email').hasError('email')">
-        You must be at least {{ form.get('email').errors['min'].min }} years old
+        You must be at least {{ form.get('age').getError('min').min }} years old
```

Comment:
That should still be email no?

Reply:
No in fact the *ngIf above might be wrong

Reply:
Fixed it. I'll merge this, and introduce the more complex changes (tabs, etc.) in a later PR.